### PR TITLE
Replace `var` to `const` or `let` in sample code

### PR DESCRIPTION
**Pull request by Korilakkuma**
The modern browsers that can use Web Audio API can also use `const` or `let`.
Therefore, legacy code like `var` avoid as much as possible.

***
<a href="https://pr-preview.s3.amazonaws.com/Korilakkuma/web-audio-api-1/pull/1999.html" title="Last updated on Jul 17, 2019, 11:16 PM UTC (b295e13)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/1999/b45b2a7...Korilakkuma:b295e13.html" title="Last updated on Jul 17, 2019, 11:16 PM UTC (b295e13)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -10377,16 +10377,16 @@ a lower bit-depth), and by quantizing in time resolution
 {{AudioWorkletProcessor}}.
 
 <pre line-numbers class="example" highlight="js" title="BitCrusher - Global Scope">
-let context = new AudioContext();
+const context = new AudioContext();
 context.audioWorklet.addModule('bitcrusher.js').then(() =&gt; {
-	let osc = new OscillatorNode(context);
-	let amp = new GainNode(context);
+	const osc = new OscillatorNode(context);
```
Comment (Korilakkuma) [inline index.bs]: It is a little problem that constructor and factory method of `AudioNode` mix in sample code. I think that it is better to unify constructor or factory method than this.

**Comment by hoch** [comment]
Marked as non substantive for IPR from ash-nazg.

File: index.bs (hunk 1 above)
Comment (hoch) [reply index.bs]: We should use the constructor where it is possible.

**Comment by hoch** [comment]
Thanks for your contribution!

**Pull request merged**